config/config.php

```diff
@@ -21,31 +21,40 @@
 
 declare(strict_types=1);
 
-$rel = '2022-01-27 12:00';
-$ver = '3.4.2';
+$stableReleaseDate = '2021-12-21 19:00';
+$stableVersion = '3.4.1';
 
-$ver_str = 'Nextcloud Client ' . $ver;
+$betaReleaseDate = '2021-11-11 12:56';
+$betaVersion = '3.4.1-rc1';
+
+$stableVersionString = 'Nextcloud Client ' . $stableVersion;
+$betaVersionString = 'Nextcloud Client ' . $betaVersion;
 
 if (version_compare($version, '3.0.3') < 0) {
 	$url = 'https://download.nextcloud.com/desktop/releases/';
-	$linux_url = $url . 'Linux/';
-	$windows_url = $url . 'Windows/';
-	$mac_url = $url . 'Mac/Installer/';
+	$stable_linux_url = $url . 'Linux/';
+	$stable_windows_url = $url . 'Windows/';
+	$stable_mac_url = $url . 'Mac/Installer/';
 } else {
-	$url = 'https://github.com/nextcloud/desktop/releases/download/v' . $ver . '/';
-	$linux_url = $url;
-	$windows_url = $url;
-	$mac_url = $url;
+	$stableUrl = 'https://github.com/nextcloud/desktop/releases/download/v' . $stableVersion . '/';
```

Comment:
I was sure we were using the nextcloud-releases org for that
so can you use `https://github.com/nextcloud-releases/desktop/releases/download/` as the base URL ?